### ISSUE TITLE
Extract libraries as npm dependencies

### ISSUE THREAD

**Issue by bguzmanrio** [MiguelCastillo/Brackets-InteractiveLinter#187]
Don't you think it would be a good idea to have require, promise, eslint and the others as npm dependencies?

**Comment by MiguelCastillo** [comment]
Yeah it would be a great idea.  I recall having issue with eslint when we tried doing it.  https://github.com/MiguelCastillo/Brackets-InteractiveLinter/pull/160

Please feel free to take a a look though.
